components/home-two/home-two-tags-selector.tsx

```
@@ -0,0 +1,219 @@
+        }
+      )}
+    >
+      <div className="flex items-center justify-between w-full px-3 py-2">
+        <div className="flex items-center gap-2">
+          {isActive && (
+            <svg
+              className="w-3 h-3 text-white"
+              fill="none"
+              stroke="currentColor"
+              viewBox="0 0 24 24"
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)